app/views/issues/new.html.erb

```diff
@@ -17,7 +17,7 @@
   <div id="issues_editor">
     <div class="note-text-inner">
       <h4 class="header-underline">Add new issue</h4>
-      <%= render 'form' %>
+      <%= render 'issues/form/form' %>
```

Comment:
Just wondering is there a reason for moving `issues/_form` to `issues/form/_form`? `issues/form` seems more intuitive to me.

Reply:
It's not aesthetic but it groups the form-building partials into one director, which was originally suggested by Daniel.